source/Playnite.DesktopApp/ViewModels/GameEditViewModel.cs

```diff
@@ -2332,9 +2348,13 @@ public void UseExeIcon()
             EditingGame.Icon = icon;
         }
 
-        public void SelectCover()
+        public void SelectCover(string path = null) 
         {
-            var path = dialogs.SelectImagefile();
+            if (path == null) 

```

Comment:
Should be checked via `IsNullOrEmpty`.

Reply:
Why? This variable may be or null or path. It not be empty anycase.